syncano/connection.py

```diff
@@ -235,12 +235,11 @@ def make_request(self, method_name, path, **kwargs):
         :raises SyncanoRequestError: if something went wrong during the request
         """
         data = kwargs.get('data', {})
+        files = data.pop('files', None)
 
-        if method_name == 'POST':
-            files = data.pop('files', {})
-        else:
+        if files is None:
```

Comment:
Are you sure that if we have files in data['files'] then we can't have files in other values?
Such scenario - is possible or not:

```
data = {
'files': {'some_file: <fileobject>'},
'filename': <someotherfileobject>
}
```

If above is true - current code will allow to pass 'data' with files objects - and this I think is what we want to avoid?